babyagi.py

```diff
@@ -116,7 +119,12 @@ def openai_call(
 ):
     while True:
         try:
-            if not model.startswith("gpt-"):
+            if model.startswith("llama"):
+                # Spawn a subprocess to run llama.cpp
+                cmd = cmd = ["llama/main", "-p", prompt]
+                result = subprocess.run(cmd, shell=True, stderr=subprocess.DEVNULL, stdout=subprocess.PIPE, text=True)
```

Comment:
This seems quite specific? And not sure masking stderr is a good idea.

Should we make this model selection a method that folks can add to?

Reply:
Yes, this should be refactored in a followup.